recipe-server/normandy/recipes/tests/api/v1/test_api.py

```diff
@@ -268,6 +268,25 @@ def test_creation_when_arguments_are_invalid(self, api_client):
             recipes = Recipe.objects.all()
             assert recipes.count() == 0
 
+        def test_creation_when_arguments_is_a_string(self, api_client):
+            action = ActionFactory(
+                name='foobarbaz',
+                arguments_schema={
+                    'type': 'object',
+                    'properties': {'message': {'type': 'string'}},
+                    'required': ['message']
+                }
+            )
+            res = api_client.post('/api/v2/recipe/', {'name': 'Test Recipe',
+                                                      'enabled': True,
+                                                      'extra_filter_expression': 'true',
+                                                      'action_id': action.id,
+                                                      'arguments': '{"message": "the message"}'})
```

Comment:
Can we avoid the hanging indent style in new code?

Reply:
yup. i hate 'em. i just copied this over from an existing test.